system-int-test/test/cmr/system_int_test/ingest/collection_misc_validation_test.clj

```diff
@@ -41,43 +40,56 @@
+             errors)))))
 
 (deftest duplicate-entry-title-test
   (testing "same entry-title and native-id across providers is valid"
     (assert-valid
-      {:entry-title "ET-1" :concept-id "C1-PROV1" :native-id "Native1"})
+      {:EntryTitle "ET-1" :Version "V1" :concept-id "C1-PROV1" :native-id "Native1"})
     (assert-valid
-      {:entry-title "ET-1" :concept-id "C1-PROV2" :native-id "Native1" :provider-id "PROV2"}))
+      {:EntryTitle "ET-1" :Version "V2" :concept-id "C1-PROV2" :native-id "Native1" :provider-id "PROV2"}))
 
   (testing "entry-title must be unique for a provider"
     (assert-conflict
-      {:entry-title "ET-1" :concept-id "C2-PROV1" :native-id "Native2"}
+      {:EntryTitle "ET-1" :Version "V3" :concept-id "C2-PROV1" :native-id "Native2"}
       ["The Entry Title [ET-1] must be unique. The following concepts with the same entry title were found: [C1-PROV1]."])))
 
 (deftest nil-version-test
   (testing "Collections with nil versions are rejected"
-    (let [concept (dc/collection-concept {:version-id nil} :iso19115)
+    (let [concept (data-umm-c/collection-concept {:Version nil} :iso19115)
           response (ingest/ingest-concept concept)]
       (is (= {:status 422
               :errors ["Version is required."]}
              response)))))
 
 (deftest field-exceeding-maxlength-warnings
   (testing "Multiple warnings returned for the fields exceeding maxlength allowed"
-    (let [collection (dc/collection-dif10
-                       {:platforms [(dc/platform {:short-name (apply str (repeat 81 "x"))})]
-                        :purpose (apply str (repeat 12000 "y"))
-                        :product (dc/product {:processing-level-id "1"})
-                        :collection-progress :complete})
+    (let [collection (data-umm-c/collection-missing-properties-dif10
+                       {:Platforms [(data-umm-c/platform {:ShortName (apply str (repeat 81 "x"))})]
+                        :Purpose (apply str (repeat 12000 "y"))
+                        :ProcessingLevel {:Id "1"}
+                        :CollectionProgress :complete})
           ingest-response (d/ingest "PROV1" collection {:format :dif10})
-          validation-response (ingest/validate-concept (dc/collection-concept collection :dif10))]
+          validation-response (ingest/validate-concept (data-umm-c/collection-concept collection :dif10))]
       (is (some? (re-find #"/Platforms/0/ShortName string.*is too long \(length: 81, maximum allowed: 80\)" (:warnings ingest-response))))
       (is (some? (re-find #"/Platforms/0/ShortName string.*is too long \(length: 81, maximum allowed: 80\)" (:warnings validation-response))))
       (is (some? (re-find #"/Purpose string.*is too long \(length: 12000, maximum allowed: 10000\)" (:warnings ingest-response))))
       (is (some? (re-find #"/Purpose string.*is too long \(length: 12000, maximum allowed: 10000\)" (:warnings validation-response)))))))
 
+(deftest multiple-warnings
```

Comment:
There are some in cmr.system-int-test.ingest.collection-umm-spec-validation-test

As for keyword validation errors - what is your question there? Keyword validation errors require the header to be passed in and there is a separate file testing all of them. I know because i break it all the time. cmr.system-int-test.ingest.collection-keyword-validation-test